vision_project/stage_2_dataset_test.py

```diff
@@ -0,0 +1,27 @@
+import os
```

Comment:
This file is just copy-paste of dataset_test. will adjust to perform tests on stage 2 when i get data

### OR

when i scheme up a good why to pseudo-test